index.js

```diff
@@ -1,41 +1,15 @@
-const toggle = document.querySelector(".toggle");
+const toggle = document.querySelector('.toggle');
 const navlinks = document.querySelector('.navlinks');
-const links = document.querySelectorAll('.link')
+const links = document.querySelectorAll('.link');
 
-toggle.addEventListener("click", () => {
-    toggle.classList.toggle("open");
-    navlinks.classList.toggle('open')
+toggle.addEventListener('click', () => {
+  toggle.classList.toggle('open');
+  navlinks.classList.toggle('open');
 });
 
 links.forEach((link) => {
-    link.addEventListener('click', () => {
-        toggle.classList.remove('open')
-        navlinks.classList.remove('open')
-    })
-})
-
-if ($(".text-slider").length == 1) {
-             
-            var typed_strings =
-                $(".text-slider-items").text();
- 
-            var typed = new Typed(".text-slider", {
-                strings: typed_strings.split(", "),
-                typeSpeed: 50,
-                loop: false,
-                backDelay: 100,
-                backSpeed: 30,
-            });
-}
-        
-$(window).scroll(function() {
-    $('.animate-on-scroll').each(function() {
-        var position = $(this).offset().top;
-        var scrollPosition = $(window).scrollTop();
-        if (position < scrollPosition + $(window).height()) {
-            $(this).addClass('animated');
-        }
-    });
+  link.addEventListener('click', () => {
+    toggle.classList.remove('open');
+    navlinks.classList.remove('open');
+  });
```

Comment:
- [ ] For some reason, the content of the whole page except the headline section and background images are not showing on page load. I tried this on two different browsers brave and firefox. Below are the screen record of both desktop and mobile versions. Kindly look into this and fix.

### Your App Behaviour (Desktop)
[app-behaviour.webm](https://user-images.githubusercontent.com/26597432/233419930-4cf007db-683a-41aa-9503-b2162dde5f0b.webm)

### Your App Behaviour (Mobile)
[mobile-app-behaviour.webm](https://user-images.githubusercontent.com/26597432/233420007-c24299f7-8f55-4f63-be7c-1c0a0f8ec181.webm)